pkg/options/options.go

```diff
@@ -23,6 +23,8 @@ const (
 	// ProviderIDFormatV1 will set provider id to machine as follows
 	// For VPC machines: ibmvpc://<cluster_name>/<vm_hostname>
 	// For Power VS machines: ibmpowervs://<cluster_name>/<vm_hostname>
+	// Deprecated: ProviderIDFormatV1 is deprecated and will be removed in a future release. see https://github.com/kubernetes-sigs/cluster-api-provider-ibmcloud/issues/1868 for more details.
```

Comment:
Sorry for late update, as you mentioned the provider id format v1 is used for vpc as well and we are deprecating it only for PowerVS, so Lets explicitly mention that its deprecated for PowerVS